### DISCUSSION TITLE
Page with form and rediec to a different view

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2743] [Help]
I have a page that has a form and i want to submit the form and then redirec to a blade view.

How can i do that?

public $data_registro = '';

    protected function getFormSchema(): array 
    {
        return [
            DatePicker::make('data_registro')
                    ->format('Y-m')
                    ->displayFormat('d/m/Y')
                    ->default(Carbon::now())
                ,
        ];
    } 

    public function submit(): void
    {
        
        $data = $this->form->getState();
        // dd($records);
        $this->tabela($data['data_registro']);

    }

    public function tabela($data)
    {
        
        $records = PestRecord::where(DB::raw("(DATE_FORMAT(data_registro,'%Y-%m'))"), $data)->get();
        // dd($records);

        return view('relatorios.tabela')->with(compact('records'));
    }

That is how im trying to do in the page class. But, when i submit the form it does not redirect to the view.

**Comment by danharrin** [comment]
https://laravel-livewire.com/docs/2.x/redirecting
